src/handlers/handler.ts

```diff
@@ -0,0 +1,19 @@
+import type { ContainerType, JSONValue } from '../types';
+
+export abstract class Handler<
```

Comment:
i think we don't use the export on definition keyword
should be:
```ts
abstract class Handler {
    ...
}
export {
    Handler
}
```

Reply:
This should be at the end `export default Handler;`, and the whole file should be called `Handler.ts`.

Reply:
This rule can help enforce it https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/exports-last.md but it's not perfect.

Reply:
Actually this https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/prefer-default-export.md! Try it @addievo.

Reply:
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/prefer-default-export.md#any is probably better.